INFO     core.jenkins_utils:jenkins_utils.py:197 running cleanup
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 403 None
DEBUG    core.jenkins_utils:jenkins_utils.py:86 failed to get page on the first try, retrying after login
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /login?from=%2F HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /j_spring_security_check HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/securityRealm/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /computer/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/credentials/store/system/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /me/my-views/view/all/submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/appearance/configSubmit HTTP/1.1" 200 134
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /user/admin/security/ HTTP/1.1" 200 None
DEBUG    conftest:conftest.py:57 Argument --remote-allow-origins=* added to the chrome
DEBUG    conftest:conftest.py:57 Argument --disable-gpu added to the chrome
DEBUG    conftest:conftest.py:57 Argument --no-sandbox added to the chrome
DEBUG    conftest:conftest.py:57 Argument --disable-dev-shm-usage added to the chrome
DEBUG    conftest:conftest.py:57 Argument --headless=new added to the chrome
DEBUG    conftest:conftest.py:57 Argument --window-size=1920,1080 added to the chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --debug --language-binding python --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Found chromedriver 136.0.7103.92 in PATH: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /usr/bin/google-chrome --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 136.0.7103.92 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 136.0.7103.92
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 136.0.7103.113
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.service:service.py:231 Started executable: `/usr/bin/chromedriver` in a child process with pid: 9880 using 0 to output -3
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:36039
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "POST /session HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "POST /session/e46fe36378bf138e98cf4880a6370571/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "GET /session/e46fe36378bf138e98cf4880a6370571/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "POST /session/e46fe36378bf138e98cf4880a6370571/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "POST /session/e46fe36378bf138e98cf4880a6370571/element/f.A810293E3E1C7CEC419B4BD286D07EC4.d.3E3641FB666C6F4DABD3BB2CD0C0BA1C.e.2/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "POST /session/e46fe36378bf138e98cf4880a6370571/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "POST /session/e46fe36378bf138e98cf4880a6370571/element/f.A810293E3E1C7CEC419B4BD286D07EC4.d.3E3641FB666C6F4DABD3BB2CD0C0BA1C.e.3/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "POST /session/e46fe36378bf138e98cf4880a6370571/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "POST /session/e46fe36378bf138e98cf4880a6370571/element/f.A810293E3E1C7CEC419B4BD286D07EC4.d.3E3641FB666C6F4DABD3BB2CD0C0BA1C.e.6/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "GET /session/e46fe36378bf138e98cf4880a6370571/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "GET /session/e46fe36378bf138e98cf4880a6370571/cookie HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 155
DEBUG    core.jenkins_utils:jenkins_utils.py:26 update_crumb: 83853ab244b7275b45b957c0d99690b015a76bf715822bf3c9860e5d0131b204
DEBUG    LoginPage:login_page.py:32 login crumb: 83853ab244b7275b45b957c0d99690b015a76bf715822bf3c9860e5d0131b204
DEBUG    MainPage:ui_element.py:53 Click on locator ('link text', 'Manage Jenkins')
DEBUG    MainPage:ui_element.py:28 waiting for element_to_be_clickable(('link text', 'Manage Jenkins'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "POST /session/e46fe36378bf138e98cf4880a6370571/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "POST /session/e46fe36378bf138e98cf4880a6370571/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "GET /session/e46fe36378bf138e98cf4880a6370571/element/f.A810293E3E1C7CEC419B4BD286D07EC4.d.350B980B55B07E155F1C6C622E1B8D48.e.53/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "POST /session/e46fe36378bf138e98cf4880a6370571/element/f.A810293E3E1C7CEC419B4BD286D07EC4.d.350B980B55B07E155F1C6C622E1B8D48.e.53/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "GET /session/e46fe36378bf138e98cf4880a6370571/url HTTP/1.1" 200 0
DEBUG    ManageJenkinsPage:ui_element.py:53 Click on locator ('xpath', "//a[@href='systemInfo']/..")
DEBUG    ManageJenkinsPage:ui_element.py:28 waiting for element_to_be_clickable(('xpath', "//a[@href='systemInfo']/.."),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "POST /session/e46fe36378bf138e98cf4880a6370571/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "POST /session/e46fe36378bf138e98cf4880a6370571/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "GET /session/e46fe36378bf138e98cf4880a6370571/element/f.A810293E3E1C7CEC419B4BD286D07EC4.d.E0164222BC336E7CDBB5E9B39024D0D3.e.90/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "POST /session/e46fe36378bf138e98cf4880a6370571/element/f.A810293E3E1C7CEC419B4BD286D07EC4.d.E0164222BC336E7CDBB5E9B39024D0D3.e.90/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "GET /session/e46fe36378bf138e98cf4880a6370571/url HTTP/1.1" 200 0
DEBUG    SystemInformationPage:ui_element.py:28 waiting for element_to_be_clickable(('css selector', '.tabBar .tab a'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "POST /session/e46fe36378bf138e98cf4880a6370571/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "POST /session/e46fe36378bf138e98cf4880a6370571/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "GET /session/e46fe36378bf138e98cf4880a6370571/element/f.A810293E3E1C7CEC419B4BD286D07EC4.d.5BE6849C3F60C744B3C37A51DF273ADE.e.102/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "POST /session/e46fe36378bf138e98cf4880a6370571/elements HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "GET /session/e46fe36378bf138e98cf4880a6370571/element/f.A810293E3E1C7CEC419B4BD286D07EC4.d.5BE6849C3F60C744B3C37A51DF273ADE.e.102/text HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "GET /session/e46fe36378bf138e98cf4880a6370571/element/f.A810293E3E1C7CEC419B4BD286D07EC4.d.5BE6849C3F60C744B3C37A51DF273ADE.e.103/text HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "GET /session/e46fe36378bf138e98cf4880a6370571/element/f.A810293E3E1C7CEC419B4BD286D07EC4.d.5BE6849C3F60C744B3C37A51DF273ADE.e.104/text HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "GET /session/e46fe36378bf138e98cf4880a6370571/element/f.A810293E3E1C7CEC419B4BD286D07EC4.d.5BE6849C3F60C744B3C37A51DF273ADE.e.105/text HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "GET /session/e46fe36378bf138e98cf4880a6370571/element/f.A810293E3E1C7CEC419B4BD286D07EC4.d.5BE6849C3F60C744B3C37A51DF273ADE.e.106/text HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:36039 "DELETE /session/e46fe36378bf138e98cf4880a6370571 HTTP/1.1" 200 0